python/proton/_handlers.py

```diff
@@ -235,8 +235,8 @@ class IncomingMessageHandler(Handler, Acking):
     events related to incoming i.e. received messages.
 
     :type auto_accept: ``bool``
-    :param auto_settle: If ``True``, settle all messages (default). Otherwise
-        messages must be explicitly settled.
+    :param auto_accept: If ``True``, accept all messages (default). Otherwise
+        messages must be explicitly accepted.
```

Comment:
Fix on your fix! I think the language from MessagingHandler is better "messages must be individually accepted or rejected"

Reply:
Yes, thanks!